src/librustdoc/passes/check_custom_code_classes.rs

```diff
@@ -79,11 +74,13 @@ crate fn look_for_custom_classes<'tcx>(cx: &DocContext<'tcx>, dox: &str, item: &
             &cx.tcx.sess.parse_sess,
             sym::custom_code_classes_in_docs,
             sp,
+            "custom classes in code blocks are unstable",
+        )
+        .note(
             // This will list the wrong items to make them more easily searchable.
             // To ensure the most correct hits, it adds back the 'class:' that was stripped.
             &format!(
-                "Found in {} block(s): class:{}",
-                tests.nb_lang_string,
+                "Found custom class(es) on code block(s): class:{}",
```

Comment:
Rust's general convention is to have lowercase errors/notes/etc., so this should be:

```suggestion
                "found custom class(es) on code block(s): class:{}",
```

But also I feel that the pluralization in the parentheses (whatever the name for that is!) is somewhat awkward. I recommend something like this:

```suggestion
                "found these custom classes: class:{}",
```

which follows the style of some other Rust errors.

Reply:
Changed for what you recommended. 😄